installer/scripts/omsadmin.sh

```diff
@@ -310,6 +313,36 @@ set_proxy_setting()
     fi
 }
 
+onboard_scom()
+{
+    echo "Onboarding SCOM"
+    create_workspace_directories "scom"
```

Comment:
As of now updation of config file is manual for us . This will require navigation to the config file location. We think that having small case scom in the path will be easier for navigation hence we prefer to leave it this way.